projects/transformers/experiments/trifecta.py

```diff
@@ -265,6 +265,7 @@ class KDLRRangeTestTrainer(LRRangeTestMixin,
     model_type="fully_static_sparse_bert",
     model_name_or_path="/mnt/efs/results/pretrained-models/transformers-local/"
     "bert_sparse_80%_trifecta_100k",
+    run_name="finetuning_bert_sparse_trifecta_100k_glue_simple"
```

Comment:
There is a rule in run.py (line 114) that if `run_name` is not given, it will automatically get the config name, so this would be redundant.